### PR TITLE
Added user icon in nav with menu dropdown

### PR DESCRIPTION
**Pull request by rozenraymond**
This PR is to solved issue #14.

Added a new nav item with user icon which will expand to show a menu list when it is clicked.
I have utilised material icon UI library for both the icon and the menu components.  

Implementation is shown in the screenshot below:

user-icon-menu

### CODE REVIEW

File: src/components/Nav/index.js [hunk 1]
```diff
@@ -25,6 +29,37 @@ const useStyles = makeStyles(theme => ({
 
 export default function Nav() {
   const classes = useStyles();
+  const [isMenuExpanded, setIsMenuExpanded] = React.useState(false);
```
Comment (lpatmo) [inline src/components/Nav/index.js]: Could we import {useState} at the very top? (e.g. `import React, { useState } from 'react'`)

Comment (rozenraymond) [reply src/components/Nav/index.js]: Yes can do! 👍

**Comment by lpatmo** [comment]
@rozenraymond Thanks for your changes! Let me know if I can merge. :)

**Comment by rozenraymond** [comment]
Yup, go ahead! :)

**Pull request merged**